.gitignore

```diff
@@ -19,6 +18,12 @@
 !.eslintrc.yaml
 !.cpplint
 
+./node_modules
```

Comment:
I'm not sure I follow the semantics of this change.
AFAIK the rules in https://github.com/nodejs/node/blob/ee3b4fd5b27d71345e9e3dcb5b1de2b6d1f91fbf/.gitignore#L5-L6
are there to un-ignore `.*` files.

There might simply be a bug in the rule at https://github.com/nodejs/node/blob/ee3b4fd5b27d71345e9e3dcb5b1de2b6d1f91fbf/.gitignore#L79
sing IIUC it should be just `/node_modules` (I think the `./` makes it global)

Reply:
another option is to put
```
!*
```
as
`tools/node_modules/.gitingore`
`tools/doc/node_modules/.gitingore`

Reply:
So `/node_modules` definitely does not work. I don't think this PR should be blocked about the `gitignore` changes by the way: the current ignore file definitely ignores to many files and this loosens that restriction to at least include the eslint files.

I updated the rule to now include all node_modules files that are not in the top level. I hope that is more what you expect?

Reply:
@refack PTAL. This should be resolved?